test/repl.jl

```diff
@@ -103,16 +103,16 @@ fake_repl() do stdin_write, stdout_read, repl
     cd(origpwd)
 
     # issue #20482
-    if !Sys.iswindows()
-        write(stdin_write, ";")
-        readuntil(stdout_read, "shell> ")
-        write(stdin_write, "echo hello >/dev/null\n")
-        let s = readuntil(stdout_read, "\n")
-            @test contains(s, "shell> ") # make sure we echoed the prompt
-            @test contains(s, "echo hello >/dev/null") # make sure we echoed the input
-        end
-        @test readuntil(stdout_read, "\n") == "\e[0m\n"
-    end
+    #if !Sys.iswindows()
+    #    write(stdin_write, ";")
+    #    readuntil(stdout_read, "shell> ")
+    #    write(stdin_write, "echo hello >/dev/null\n")
+    #    let s = readuntil(stdout_read, "\n")
+    #        @test contains(s, "shell> ") # make sure we echoed the prompt
+    #        @test contains(s, "echo hello >/dev/null") # make sure we echoed the input
+    #    end
+    #    @test readuntil(stdout_read, "\n") == "\e[0m\n"
+    #end
```

Comment:
Note: these disabled tests are left as motivation for someone to finish the implementation of https://github.com/JuliaLang/julia/issues/20401 🙂

Reply:
Do these tests not pass anymore?

Reply:
With the current deprecation, they're partially broken. Hence the motivation for someone to remove the deprecation and finish implementing the #20401